### DISCUSSION TITLE
Chaskiq not showing integrations in production mode.

### DISCUSSION THREAD

**Discussion by niflaot** [chaskiq/chaskiq#973] [Q&A]
Hello. I have tried to install Chaskiq following the production guide, I did the following:

- Defined th environment variables
- Set up docker containers with redis, postgre, rpc, anycable, sidekiq and chaskiq with the same env variables
- Seeding database and generating admin user
- Configured with SMTP and MinIO

Everything is working fine. However, I haven't figured out how to add Twilio, Messenger or other plugins. It just shows ArticleSearch, ContactFields, ContentShowcase and Inbox Sections.

**Comment by michelson** [comment]
you can get the plugins at the appstore.chaskiq.io
